skaffold.yaml

```diff
@@ -16,9 +19,6 @@ manifests:
   kustomize:
     paths:
       - config/dev
-  ## Deploy the network attachment definition CRD.
-  rawYaml:
-  - hack/crds/*
```

Comment:
Moved this to the Makefile. Having it here means it automatically deletes the CRDs when skaffold is torn down - seems unnecessarily destructive. In the makefile it's a one-shot install of the CRD.